flink-core/src/main/java/org/apache/flink/util/NetUtils.java

```diff
@@ -120,6 +120,18 @@ private static URL validateHostPortString(String hostPort) {
         }
     }
 
+    /**
+     * Converts an InetSocketAddress to a URL. This method assigns the "http://" schema to the URL
```

Comment:
So the existing implementation does not support https at all? I know that you are going to add support for https scheme later, just wanted to clarify the current state for host:port ?